dartagnan/src/main/java/com/dat3m/dartagnan/parsers/program/visitors/VisitorLitmusVulkan.java

```diff
@@ -288,10 +290,23 @@ private void tagChecker(AbstractEvent e, Boolean atomic, String mo, String scope
             throw new ParsingException("Visible avvisSemantic must be acquire mo");
         }
 
+        // ----------------------------------------------------------------------------------------------------------------
         // Add tags
         e.addTags(mo, avvisSemantic);
         if (atomic || e instanceof VulkanRMW) {
```

Comment:
You can directly add the tag to every `VulkanRMW` event and get rid of the second part of the check

Reply:
Instead of adding this in the parser / visitor, you can directly add it in the constructor of `VulkanRMW.java`

Reply:
Moved